com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.cs

```diff
@@ -49,7 +49,7 @@ enum Expandable
             RTGIQuality = 1 << 33
         }
 
-        static readonly ExpandedState<Expandable, HDRenderPipelineAsset> k_ExpandedState = new ExpandedState<Expandable, HDRenderPipelineAsset>(Expandable.CameraFrameSettings | Expandable.General, "HDRP");
+        static readonly ExpandedState<Expandable, HDRenderPipelineAsset> k_ExpandedState = new ExpandedState<Expandable, HDRenderPipelineAsset>(Expandable.VirtualTexturing | Expandable.General, "HDRP");
```

Comment:
Why VirtualTexturing here? Shouldn't it be just "General"?

Reply:
yes sorry i was trying something to understand how it works. will be fixed in upcoming commit

Reply:
fixed in last commit